h/api/search/query.py

```diff
@@ -98,3 +118,16 @@ def _match_clause_for_uri(uristr):
             "should": matchers
         }
     }
+
+
+def _filter_clause_for_uri(uristr):
```

Comment:
I might suggest calling this `_term_clause_for_uri` as the other is called `_match_clause_for_uri` and both are ignorant of whether they're being added to a filter or a query. Here, `filters` is really just something more like `clauses` and it's the caller that's choosing to put it into a filter.